MapsetVerifier.Checks/Taiko/Timing/CheckKiaiConsistency.cs

```diff
@@ -0,0 +1,80 @@
+﻿using MapsetVerifier.Framework.Objects;
+using MapsetVerifier.Framework.Objects.Attributes;
+using MapsetVerifier.Framework.Objects.Metadata;
+using MapsetVerifier.Parser.Objects;
+
+using static MapsetVerifier.Checks.Utils.TaikoUtils;
+
+namespace MapsetVerifier.Checks.Taiko.Timing
+{
+    [Check]
+    public class CheckKiaiConsistency : GeneralCheck
+    {
+        public override CheckMetadata GetMetadata() =>
+            new BeatmapCheckMetadata()
+            {
+                Author = "SN707, Nostril",
+                Category = "Compose",
+                Message = "Kiai inconsistencies",
+                Modes = [Beatmap.Mode.Taiko],
+                Documentation = new Dictionary<string, string>()
+                {
+                    {
+                        "Purpose",
+                        @"
+                    Pointing out kiai inconsistencies between difficulties."
+                    },
+                    {
+                        "Reasoning",
+                        @"
+                    Kiais should generally be consistent across difficulties (except in GDs)."
```

Comment:
I don't know the reason why consistency is better in Taiko, but something better should be written here.